docs/application/toc_all.md

```diff
@@ -916,12 +916,13 @@
 #### [Activity Recognition](/application/native/guides/location-sensors/activity.md)
 #### [Gesture Recognition](/application/native/guides/location-sensors/gesture.md)
 
-### Text Input and Voice
+### Text Input, Gesture and Voice
```

Comment:
Actually, I thought 'Gesture' is in ambiguous position, because it is neither text input or voice.
So, I input 'Gesture' in the heading.
I'm not sure that changing the heading is acceptable, like 'User Interaction Experience' or other title.
It seems that the discussion is necessary @_@

Reply:
@mijin85cho, please let us know your view as well.
In that case we can make an exception and change to:

### Text Input, Gesture, and Voice

Reply:
Also I have a question at adding Gesture here
as we have native guide page for gesture like https://docs.tizen.org/application/native/guides/location-sensors/gesture/.

I can see the new Gesture page is different with previous one, but is there any reason that the new Gesture should be placed separately, at here?

Reply:
> Also I have a question at adding Gesture here
> as we have native guide page for gesture like https://docs.tizen.org/application/native/guides/location-sensors/gesture/.
> 
> I can see the new Gesture page is different with previous one, but is there any reason that the new Gesture should be placed separately, at here?

As you mentioned, there is another Gesture in location-sensors (managed by Context team). This is going to be deprecated after 2 Tizen versions. And, our part (Interaction team) will take a charge of Gesture APIs. Actually, 'text-input' belongs to our part. So, I also added 'capi-ui-gesture.md' into 'text-input', but it seems a little bit weird... @_@ If possible, I think it will be better to change 'text-input' to others, like 'user-interaction' , etc. How about you?? Please give me your opinion.

Reply:
As we have guide for Web and .NET also, we structured the docs to have similar structure between Native, Web and .NET.

The category title "Text input" is also shared between .NET and Web. So if we change the "Text input" title under Native to "user-interaction", we should consider the category title of Web and .NET also.


**.NET :**
![image](https://user-images.githubusercontent.com/9958374/95933633-03bb4b00-0e0a-11eb-8fdb-40c7f82aade5.png)

**Web :**
![image](https://user-images.githubusercontent.com/9958374/95933657-133a9400-0e0a-11eb-8c04-38b62e7fa7ed.png)

**Native :**
![image](https://user-images.githubusercontent.com/9958374/95933685-2188b000-0e0a-11eb-831a-e586d49d516f.png)

Changing to "user-interaction" seems good idea, but we should consider it. So for now I think the first modification (adding "Gesture") is the second best plan.

Reply:
> As we have guide for Web and .NET also, we structured the docs to have similar structure between Native, Web and .NET.
> 
> The category title "Text input" is also shared between .NET and Web. So if we change the "Text input" title under Native to "user-interaction", we should consider the category title of Web and .NET also.
> 
> **.NET :**
> ![image](https://user-images.githubusercontent.com/9958374/95933633-03bb4b00-0e0a-11eb-8fdb-40c7f82aade5.png)
> 
> **Web :**
> ![image](https://user-images.githubusercontent.com/9958374/95933657-133a9400-0e0a-11eb-8c04-38b62e7fa7ed.png)
> 
> **Native :**
> ![image](https://user-images.githubusercontent.com/9958374/95933685-2188b000-0e0a-11eb-831a-e586d49d516f.png)
> 
> Changing to "user-interaction" seems good idea, but we should consider it. So for now I think the first modification (adding "Gesture") is the second best plan.

Thank you for your comment. Then, for now, I also think that adding "Gesture" is the best. Then, is it OK to add "Gesture" with a comma in the heading ??
I think Ms. Gupta thinks adding a comma in the heading is weird.
Could you check adding a comma is OK or not??

Reply:
Hello, thanks for checking. Usually we don't use commas in the heading, but for this case, I think we can make an exception.
So let's apply the way @surbhig12 also suggested above

\### Text Input, Gesture, and Voice

Reply:
> Hello, thanks for checking. Usually we don't use commas in the heading, but for this case, I think we can make an exception.
> So let's apply the way @surbhig12 also suggested above
> 
> ### Text Input, Gesture, and Voice

Thank you for your comment. I changed it with your recommendation. Please review this again.
Thank you!!